handbook/company/pricing-features-table.yml

```
@@ -587,6 +661,8 @@
 - industryName: Low-level MDM commands for macOS, iOS/iPadOS*, and Windows (e.g. remote restart)
   documentationUrl: https://fleetdm.com/docs/using-fleet/mdm-commands
   tier: Free
+  jamfProHasFeature: macOnly
```

```suggestion
  jamfProHasFeature: macOnly
```

iOS and iPadOS too

---

@noahtalerman What do you think about using `appleOnly` for these? 

```suggestion
  jamfProHasFeature: appleOnly
```

---

Makes sense to me! Good thinking.